qtp_biom/summary.py

```diff
@@ -84,13 +56,62 @@ def generate_html_summary(qclient, job_id, parameters, out_dir):
          sample_count_summary['Median count']),
         ("<b>Mean count:</b> %d<br/>" %
          sample_count_summary['Mean count']),
-        '<br/><hr/><br/>',
-        '<img src = "%s"/>' % uri
-    ]
+        '<br/><hr/><br/>']
+
+    if sample_count_summary['Minimum count'] == sample_count_summary[
+            'Maximum count']:
+        artifact_information.append(
+            "All the samples in your BIOM table have %d sequences"
```

Comment:
Only minor addition would be to mention that no plot will be displayed, something like: 

All the samples in your BIOM table have XXX sequences, no plot will be shown below.